test/functional/viml/errorlist_spec.lua

```diff
@@ -0,0 +1,30 @@
+local helpers = require('test.functional.helpers')
+
+local clear   = helpers.clear
+local command = helpers.command
+local eq      = helpers.eq
+local eval    = helpers.eval
```

Comment:
As usual I would suggest replacing `eval` with `helpers.funcs.setqflist(…)` and `helpers.curwinmeths.get_var('quickfix_title')`, there is no need to invoke expression parser.

Also it is not good idea to align equal signs because it needs realignment once somebody adds something more lengthy like `curwinmeths` in the example. (I sometimes entertain myself arranging variables by their length which also may look good yet does not require realigning, but not rearranging though. And sometimes, not always.)